src/EditorFeatures/CSharpTest/Completion/CompletionProviders/Snippets/AbstractCSharpConditionalBlockSnippetCompletionProviderTests.cs

```diff
@@ -467,5 +467,36 @@ void M(bool arg)
 
             await VerifyItemIsAbsentAsync(markupBeforeCommit, ItemToCommit);
         }
+
+        [WpfFact]
+        public async Task CorrectlyDealWithLeadingTriviaInInlineSnippetTest()
+        {
+            var markupBeforeCommit = """
+                class Program
+                {
+                    void M(bool arg)
+                    {
+                        // comment
```

Comment:
💡 Consider adding tests for inline snippets as top-level statements at the end of the file, again with various trivia